sdk/src/main/java/com/silanis/esl/sdk/internal/UrlTemplate.java

```diff
@@ -164,6 +168,9 @@
     // Account Signing Themes Service
     public static final String ACCOUNT_SIGNING_THEME_PATH = "/account/signingThemes";
 
+    // Account Signing Logos Service
+    public static final String ACCOUNT_SIGNING_LOGO_PATH = "/account/admin/signingLogos";
+
```

Comment:
This part has already been modified and doesn't need to be modified. Please rebase the dev branch.